src/app/models/node-agent-sku.ts

```diff
@@ -1,19 +1,20 @@
-import { ListProp, Model, Prop, Record } from "@batch-flask/core";
-import { List } from "immutable";
+import { Model, Prop, Record } from "@batch-flask/core";
 import { ImageReference, ImageReferenceAttributes } from "./image-reference";
 
-export interface NodeAgentSkuAttributes {
-    id: string;
-    verifiedImageReferences: ImageReferenceAttributes[];
+export interface ImageInformationAttributes {
+    verificationType: string;
+    nodeAgentSKUId: string;
     osType: string;
+    imageReference: ImageReferenceAttributes;
 }
 
 /**
  * Class for displaying Batch Node Agent SKU information.
  */
 @Model()
-export class NodeAgentSku extends Record<NodeAgentSkuAttributes> {
-    @Prop() public id: string;
-    @ListProp(ImageReference) public verifiedImageReferences: List<ImageReference> = List([]);
+export class ImageInformation extends Record<ImageInformationAttributes> {
```

Comment:
Is `ImageInformation` the correct name(In the service) or is it `SupportedImage` like the url?

Reply:
ImageInformation weirdly enough https://docs.microsoft.com/en-us/rest/api/batchservice/account/listsupportedimages#accountlistsupportedimagesresult